compiler/Makefile

```diff
@@ -10,5 +14,8 @@ v.c:
 test:
 	find .. -name '*_test.v' | xargs v {}
 
+install: v
+	cp -f ./v $(PREFIX)/bin
```

Comment:
It might be better to allow the user to change the bin dir at compile time, but default to `/bin`. For example, given that you have the following macros:

```
PREFIX ?= /usr/local
BINDIR ?= /bin
```

You could write this as:

```
cp -f ./v $(PREFIX)$(BINDIR)
```